docs/UserGuide.md

```diff
@@ -100,32 +100,46 @@ ___
 
 ### Adding a new client: `add`
 
-This command adds a new client to your client list.
+This command adds a new client to your client list. All fields except the tag field are compulsory.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]… [k/NEXTOFKIN]`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS k/NEXTOFKIN d/DESCRIPTION [t/TAG]…`
 
```

Comment:
Should have a bullet point list of parameters constraints, name must be unique, and so on

Reply:
The part where "all parameters of the add command is compulsory" above should be moved to parameter constraints